test/acceptance/providers/log-action.provider.acceptance.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback-next-extension-starter
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {Client, createClientForHandler, sinon} from '@loopback/testlab';
+import {inject} from '@loopback/context';
+import {Application} from '@loopback/core';
+import {
+  get,
+  param,
+  SequenceHandler,
+  FindRoute,
+  ParseParams,
+  InvokeMethod,
+  Send,
+  Reject,
+  RestBindings,
+  ParsedRequest,
+  ServerResponse,
+  RestServer,
+  RestComponent,
+} from '@loopback/rest';
+import {LogComponent, ExtensionStarterBindings, LogFn} from '../../..';
+
+const coreSequenceActions = RestBindings.SequenceActions;
+
+describe('Logging (acceptance)', () => {
+  let app: Application;
+  let server: RestServer;
+
+  // tslint:disable-next-line:no-any
+  let spy: any;
+  beforeEach(() => {
+    spy = sinon.spy(console, 'log');
+  });
+
+  beforeEach(createApplication);
+  beforeEach(createController);
+  beforeEach(getServerFromApp);
+  beforeEach(createSequence);
+
+  afterEach(() => {
+    spy.restore();
+  });
+
+  it('logs request information', async () => {
+    const client: Client = createClientForHandler(server.handleHttp);
+    await client.get('/?name=John').expect(200, 'Hello John');
+    const expectedLog = '100.02ms: /?name=John : (John) => Hello John';
+    sinon.assert.calledWith(spy, sinon.match(expectedLog));
```

Comment:
If you are expecting the exact string, then I think you should omit `sinon.match`.

```ts
sinon.assert.calledWith(spy, expectedLog);
```

Unless I am missing something?